assignment5.py

```diff
@@ -2,9 +2,71 @@
 from utilityModule import *
 
 
+def get_user_ratings(dataframe):
+    user_ratings_dict = {}
+    for index, row in dataframe.iterrows():
+        user_id = row["userId"]
+        movie_id = row["movieId"]
+        rating = row["rating"]
+        if user_id not in user_ratings_dict:
+            user_ratings_dict[user_id] = {}
+        user_ratings_dict[user_id][movie_id] = rating
+    return user_ratings_dict
```

Comment:
I think this function could be simplified using some of the built-in features of pandas. I may try to do it later, for now it is good.